Utilities/DataFlow/include/DataFlow/TimeFrameValidatorDevice.h

```diff
@@ -0,0 +1,33 @@
+#ifndef ALICEO2_TIMEFRAMEVALIDATOR_H_
+#define ALICEO2_TIMEFRAMEVALIDATOR_H_
+
+#include "O2Device/O2Device.h"
+
+namespace AliceO2 {
+namespace DataFlow {
+
+/// A validating device for time frame data (coming from EPN)
+class TimeframeValidatorDevice : public Base::O2Device
```

Comment:
There is a mismatch in the header file name, should be `TimeframeValidatorDevice.h` instead of `TimeFrameValidatorDevice.h`